introduction/site_survey.md

```diff
@@ -0,0 +1,353 @@
+| ---- | ----- |
+| root_days | |
+| int_days | |
+| root_cn | |
+| int_cn | |
+| Is a site (external) CA available? | |
+| Is a site (external) CA private key available? | |
+| Is a site (external) CA certificate available? | |
+
+   > Note: Outside of a new installation of the CSM software, there is currently no supported method to rotate (change) the platform CA. Please ensure validity periods are set accordingly for external CAs you use in this process. The ability to rotate CAs is anticipated as part of a future release.
+
+
+| (Optional) LDAP Settings | Value |
+| ---- | ----- |
+| Is  a site LDAP server available? | |
+| First site LDAP server | |
+| First site LDAP server port | |
+| (Optional) Second site LDAP server | |
+| (Optional) Second site LDAP server port | |
+| (Optional) Third site LDAP server | |
+| (Optional) Third site LDAP server port | |
+| site LDAP ldapSearchBase | |
+| site LDAP localRoleAssignments | |
+
+   > Note: Setting forwardZones is needed if the site LDAP server is specified via a hostname rather than an IP address. See [Prepare Site Init](../install/prepare_site_init.md)
+
+
+<a name="application_nodes"></a>
+## 10.  Application Nodes
+
+Each application node can have specific information about it.  Some application nodes have additional network connections to Ethernet or InfiniBand besides the CAN below.
+
+The only pre-defined Application node SHCD prefix and subrole is for UAN (User Access Node).
+
+| Each Application Node | Value |
+| ---- | ----- |
+| BMC or iLO username | |
+| BMC or iLO password | |
+| SHCD Prefix | |
+| Subrole | |
+| Hostname alias(es) | |
+| Is CAN enabled for this node? | |
+| CAN IP address | |
+| CAN Default route/gateway | |
+| CAN Netmask | |
+| Network Interface connected to CAN | | 
+| Is network interface (net1) enabled for this node?? | |
+| Net1 bootproto | |
+| Net1 device | |
+| Net1 IP address | |
+| Net1 startmode | |
+| Net1 ifroute route(s) | |
+| Net1 ifrule rules | | 
+| Is network interface (net2) enabled for this node?? | |
+| Net2 bootproto | |
+| Net2 device | |
+| Net2 IP address | |
+| Net2 startmode | |
+| Net2 ifroute route(s) | |
+| Net2 ifrule rules | | 
+
+
+Common settings for all User Access Nodes (UANs).  These could be set the same for all Application nodes rather than being set only for UANs.
+
+| Common UAN settings | Value |
+| ---- | ----- |
+| UAN global route(s) | |
+| UAN external DNS searchlist | |
+| UAN first external DNS server | |
+| UAN second external DNS server | |
+| UAN third external DNS server | |
+| UAN external DNS options | |
+| UAN LDAP enabled for login? | |
+| UAN LDAP domain | |
+| UAN LDAP search_base | |
+| UAN LDAP server(s) | |
+| UAN LDAP chpass_uri | |
+| UAN AD groups | |
+| UAN PAM modules | |
+
+<a name="filesystems"></a>
+## 11.  Filesystems
+
+For clients of a filesystem, there is some common data needed to be able to mount it.
+The filesytem type (fstype) could be Lustre, SpectrumScale (GPFS), or NFS.
```

Comment:
Change filesytem to filesystem.

Reply:
You may want to wait until he fixes the linter errors that are being reported, as those will catch some of the spelling/terminology errors.

Reply:
FIxed